### PR TITLE
boards/thingy53: add sensors support

### PR DESCRIPTION
**Pull request by raiden00pl**
## Summary
boards/thingy53: add sensors support
Add support for sensros: ADXL362, BH1749NUC, BMI270, BMM150

### CODE REVIEW

File: boards/arm/nrf53/common/include/nrf53_adxl362.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+/****************************************************************************
+ * Public Functions Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf53_adxl362_initialize
+ *
+ * Description:
+ *   Initialzie and register the ADXL362 as uorb sensor
```
Comment (hartmannathan) [inline boards/arm/nrf53/common/include/nrf53_adxl362.h:38]:
```suggestion
 *   Initialize and register the ADXL362 as uorb sensor
```

File: boards/arm/nrf53/common/include/nrf53_bmi270.h [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+/****************************************************************************
+ * Public Functions Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf53_bmi270spi_initialize
+ *       nrf53_bmi270i2c_initialize
+ *
+ * Description:
+ *   Initialzie and register the BMI270 as uorb sensor
```
Comment (hartmannathan) [inline boards/arm/nrf53/common/include/nrf53_bmi270.h:39]:
```suggestion
 *   Initialize and register the BMI270 as uorb sensor
```

File: boards/arm/nrf53/common/src/nrf53_adxl362.c [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf53_adxl362_init
+ *
+ * Description:
+ *   Initialzie and register the ADXL362 device
```
Comment (hartmannathan) [inline boards/arm/nrf53/common/src/nrf53_adxl362.c:44]:
```suggestion
 *   Initialize and register the ADXL362 device
```

File: boards/arm/nrf53/common/src/nrf53_bmi270.c [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf53_bmi270spi_initialize
+ *       nrf53_bmi270i2c_initialize
+ *
+ * Description:
+ *   Initialzie and register the BMI270 character device as 'devpath'
```
Comment (hartmannathan) [inline boards/arm/nrf53/common/src/nrf53_bmi270.c:49]:
```suggestion
 *   Initialize and register the BMI270 character device as 'devpath'
```

File: boards/arm/nrf53/thingy53/src/nrf53_sensors.c [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf53_sensors_init
+ *
+ * Description:
+ *   Initialzie on-board sensors
```
Comment (hartmannathan) [inline boards/arm/nrf53/thingy53/src/nrf53_sensors.c:76]:
```suggestion
 *   Initialize on-board sensors
```

File: boards/arm/nrf53/thingy53/src/thingy53.h [hunk 6]
```diff
@@ -159,6 +161,16 @@
 
 int nrf53_bringup(void);
 
+/****************************************************************************
+ * Name: nrf53_sensors_init
+ *
+ * Description:
+ *   Initialzie on-board sensors
```
Comment (hartmannathan) [inline boards/arm/nrf53/thingy53/src/thingy53.h:168]:
```suggestion
 *   Initialize on-board sensors
```

File: boards/arm/nrf53/thingy53/src/nrf53_bringup.c [hunk 7]
```diff
@@ -260,6 +258,16 @@ int nrf53_bringup(void)
     }
 #endif
 
+  /* Initialzie on-board sensors */
```
Comment (hartmannathan) [inline boards/arm/nrf53/thingy53/src/nrf53_bringup.c:261]:
```suggestion
  /* Initialize on-board sensors */
```

**Review by hartmannathan — COMMENTED** [review]
Commit log: s/sensros/sensors/ (at "Add support for sensros...")

**Review by acassis — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]

**Pull request merged**